sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/Distribute.java

```diff
@@ -1039,6 +1075,12 @@ synchronized void storeRedirectionParameters(@NonNull String requestId, @NonNull
             processDistributionGroupId(distributionGroupId);
             AppCenterLog.debug(LOG_TAG, "Stored redirection parameters.");
             cancelPreviousTasks();
+
+            /* Do not proceed if automatic check for update is disabled and manual check for update has not been called. */
+            if (mAutomaticCheckForUpdateDisabled && !mManualCheckForUpdateRequested) {
```

Comment:
Might not be needed because this would only matter if we wanted to allow the flag state to be recognized in the middle of the authenticate/http call flow